.travis.yml

```diff
@@ -1,4 +1,4 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.12"
+  - "10"
```

Comment:
This should be done as a specific PR IMO, but apart from that this PR looks good ^_^

Reply:
OK, I did it as three commits in one PR, but I can split it out as one PR per commit, no problem!